### DISCUSSION TITLE
dynamic perimeter order

### DISCUSSION THREAD

**Discussion by Nukle0n** [OrcaSlicer/OrcaSlicer#2196] [Ideas]
Outside/Inside/Infill is more accurate but it breaks on overhangs, so unless you're printing something that's basically a pyramid you can't use it. I think the idea gets floated now and then but it'd be really neat with something that uses the already calculated overhang percentages to force the perimeters for overhangs back to Inside/Outside/Infill.

**Comment by harriettubgirl** [comment]
I really like this idea, hopefully it's a potential to add this!
